x-pack/plugins/reporting/server/routes/common/jobs/job_management_pre_routing.ts

```diff
@@ -44,15 +39,6 @@ export const jobManagementPreRouting = async (
   }
 
   const { jobtype } = doc;
-  if (!jobTypes.includes(jobtype)) {
-    return res.forbidden({
```

Comment:
This is a bug that prevents users from downloading past reports which are no longer supported in implementation